README.md

```diff
@@ -1,16 +1,49 @@
 # ansible-galaxy-subdomains
 
-Sets up the fancy subsite CSS/welcome pages and tool sections on a per-subdomain basis like EU does
+Sets up the subdomain static content and themes.  
+(not fully serious) [Presentation on the topic](https://docs.google.com/presentation/d/1N1u1b0AqJ-ZVz8qKO5YxkwUSjSR5HwnuSxTsj6wAevE/)
+
+:warning: Important: All subdomains need a themes.yml file in a location set in
+~~~yaml
+subdomains_themes_files: "{{ lookup('fileglob', 'files/galaxy/config/themes/*.yml', wantlist=True) }}"
+~~~
+the name **must** match the subdomain name.  
+E.g. for example.usegalaxy.eu you need  
+~~~
+files/galaxy/config/themes/example.yml
+~~~
+Otherwise the subdomains may not be considered by the role.  
+
+:warning:**NGINX**: Make sure that Galaxy serves all subdomains (e.g. by omit `sever_name` and set `proxy_set_header Host $http_host;` under `location /`)
+
+## The Role can do the following:
+### Create static directories per subdomain when `subdomains_serve_static: true`
+- Creates a full depth copy of Galaxy's static dir for each subdomain
+- Copy subdomain specific static files, like `favicon.ico` or images. You need to create a directory under `subdomains_ansible_file_path` for each subdomain (and named like the subdomain) with subdirectories for each value in `subdomains_static_keys`, if you have a file in it (can be omitted otherwise). The file structure could look like the following:
+~~~bash
+files
+└── galaxy
+    ├── static
+    │   └── climate
```

Comment:
Could you maybe add a second subdomain here so it's more obvious?

Q: would it not be better for contributors to reverse the logic and have top-level `climate` with sub-dirs like `static`, `themes` .... this way we only request one folder from subdomain maintainers.

Reply:
I had them separately because the themes would fit more in files/../config, wheras the static files are not inherently configuration. I think it would not decrease amount of folders in total to create (even increase it)
I do not see the advantage yet, but I can change it in the way you want

Reply:
Leave it like it is if you are not convinced and see how contributors handle it later.

Reply:
I have to say I agree with @bgruening here, a structure like

```
themes
- climate
  - theme.yaml
  - images/
  - welcome.html
```

would be more user friendly than needing to add things in two places. I'm not sure how difficult that would be to query though from the Ansible side.

Reply:
I pushed a commit, it has now a structure like:
subdomains(or whatever)
 - climate
   - static
     - dist
     - images
     - ...
   - themes
     - climate.yml